website/source/docs/provisioners/shell-local.html.md

```diff
@@ -108,6 +108,14 @@ Optional parameters:
     **Important:** If you customize this, be sure to include something like the
     `-e` flag, otherwise individual steps failing won't fail the provisioner.
 
+-  `onlyon` (array of strings) - This is an array of 
+    [runtime operating systems](https://golang.org/doc/install/source#environment)
+    where `shell-local` will only run on. This allows you to run `shell-local` *only* 
+    on specific compatible operating systems. If specified, shell-local will only
+    execute if runtime operating system is in the list; otherwise it will skip the 
+    `shell-local` command. The default behavior is for `shell-local` to run if 
```

Comment:
How about "By default, `shell-local` will always run if `only_on` is not set."

Reply:
Love it, fixing. I was struggling with the words on this 👍